ui/hooks/metamask-notifications/useNotifications.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { renderHook, act } from '@testing-library/react-hooks';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import type { Store } from 'redux';
+import * as actions from '../../store/actions';
+import { MetamaskNotificationsProvider } from '../../contexts/metamask-notifications/metamask-notifications';
+import {
+  useCreateNotifications,
+  useDisableNotifications,
+} from './useNotifications';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+jest.mock('../../store/actions', () => ({
```

Comment:
Personal pref - I like `.spyon`/spies over mocks. That way we get strongly typed mocks.